styles/main.css

```diff
@@ -0,0 +1,184 @@
+/* CSS HEX */
+:root {
+--light-cyan: #e0fbfcff;
+--powder-blue: #c2dfe3ff;
+--pewter-blue: #9db4c0ff;
+--cadet: #5c6b73ff;
+--gunmetal: #253237ff;
+--light-cyan-2: #d1edf0ff;
+
+}
+
+
+
+body {
+    font-family: 'Fredoka One', sans-serif;
+    background-color: var(gunmetal);
+    height: 100%;
+    
+}
+
+main {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 10px; 
+    margin-top: 10px;
+    margin-bottom: 5px;
+    
+}
+
+
+header {
+    text-align: center;
+    background-color: var(--light-cyan-2);
+    padding-bottom: 5px;
+    
+}
+
+h1 {
+    font-size: 3em;
+}
+
+
+
+
+h2 {
+    font-size: 1.8rem;
+    font-weight: bold;
+}
+
+
+content {
+    width: 80%;
+}
+
+p {
+    margin: var(--gutter);
+}
+  
+footer {
+    position: relative;
+    text-align: center;
+    font-size: small;
+    height: 2em;
+    width: 100%;
+    bottom: 0;
+    background-color: var(--light-cyan-2);
+    box-shadow: var(--shadow);
+}
+  
+
+h3 {
+    font-size: 2em;
+    display: flex;
+    justify-content: center;
+    align-self: center;
+}
+
+.quiz-area {
+    background-color: var(--powder-blue);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    padding-top: 15px;
+    
+}
+
+button {
+    width: 150px;
+    height: 100px; 
+    border-radius: 50px;
+    margin-top: 20px;
+    font-size: large;
+    font-weight: bold;
+}
+
+.factors {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-color: var(--powder-blue);
+}
+
+.basics {
+    background-color: var(--powder-blue);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+
+}
+
+.image {
+    background-color: violet;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    background-color: var(--light-cyan);
+}
+
+.family {
+    background-color: var(--pewter-blue);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    
+}
+
+.species {
+    background-color: var(--light-cyan);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+}
+
+    
```

Comment:
Just need one line of space in between :)

Reply:
updated! Just want to make sure that this means there needs to be a line of space before every end bracket in css?

Reply:
Close! Here's an example! :) 

```css 
h1 {
    color: yellow;
}

p {
    background-color: red;
}
```